README.md

```diff
@@ -70,15 +70,70 @@ and to stop all the processes you can do
 stopObserveAll
 ``` -->
 
-# How to package for deployment
+# Deployment
 
-If you haven't already, create a symlink `jre` in the project directory that points to the Linux JRE you want bundled with the deployment.
+Deployment is done via Docker images.
 
-To deploy, run in `sbt`:
+## Staging
+
+Make sure you have both `docker` and `heroku` CLIs installed and working.
+
+If you haven't already, run
+
+```
+heroku login
+heroku container:login
+```
+
+This will give your system access to Heroku's Docker registry.
+
+To deploy to Heroku, run in `sbt`:
+
+```
+deploy_observe_server/docker:publish
```

Comment:
Is it possible to deploy locally?

Reply:
Yes! You can `docker:publishLocal`. To run the container you just need to pass `SSO_SERVICE_JWT` (for staging SSO).

In fact, `publish` does a `publishLocal` folowed by a `push`.